src/main/java/school/lemon/changerequest/java/introduction/hw2/Task2.java

```diff
@@ -3,6 +3,17 @@
 
 public class Task2 {
     public static String firstTwo(String s) {
-        return "";
+
+        if (s.length()<2)
```

Comment:
What about `null` values ? you'll get `NullPointerException` when you'll try to use `.length()` method on `null`-valued string `s`.
Add additional check for `null`